interfaces.go

```diff
@@ -160,6 +160,37 @@ var sensitiveHeaders = map[string]struct{}{
 	"Cookie":              {},
 	"X-Forwarded-For":     {},
 	"X-Real-Ip":           {},
+	"Password":            {},
+	"Passwd":              {},
+	"Secret":              {},
+	"Api-Key":             {},
+	"Apikey":              {},
+	"Auth":                {},
+	"Credentials":         {},
+	"Mysql-Pwd":           {},
+	"Privatekey":          {},
+	"Private-Key":         {},
+	"Token":               {},
+	"Ip-Address":          {},
+	"Session":             {},
+	"Csrftoken":           {},
+	"Sessionid":           {},
+	"Remote-Addr":         {},
+	"X-Csrftoken":         {},
+	"Set-Cookie":          {},
+	"X-Api-Key":           {},
+	"Aiohttp-Session":     {},
+	"Connect.sid":         {},
```

Comment:
Isn't this Express specific?

Reply:
let's keep one conversation here for all these comments: https://github.com/getsentry/sentry-go/pull/1008#discussion_r2092787497